tests/15-rpl-classic/js/12-rpl-rank-error.js

```diff
@@ -0,0 +1,15 @@
+TIMEOUT(600000, log.testFailed());
```

Comment:
I don't think the second parameter is required, a timed out test should fail.

Reply:
Fixed in #2548